### DISCUSSION TITLE
Page not found - step 2 POST

### DISCUSSION THREAD

**Discussion by infxmousrogue** [timdorr/tesla-api#385] [Q&A]
Hi all, when trying the new OAuth method for the TeslaAPI as documented [here](https://tesla-api.timdorr.com/api-basics/authentication).

I'm running into a problem where when POSTing step 2 it returns a "Page not found" Why does this occur? I am using the same cookie as with the step 1 GET request. Copied all the other hidden fields from the get request and it's still not working. Did anyone run into the same issue?

Picture of my params:
![image](https://user-images.githubusercontent.com/54747969/119490396-12cd8880-bd5d-11eb-95ed-c16a5da27e25.png)

Picture of my body params:
![image](https://user-images.githubusercontent.com/54747969/119490338-05180300-bd5d-11eb-9c68-287ce6ddfdd2.png)

Body parameters of credential and identity are not in the picture for obvious reasons but they're also there.

I am currently testing this through Postman to check if I understand everything correctly and if it works. When I verify everything I will be writing the script in Swift.

Every request after setting up the same cookies as the get request results in a page not found. When deleting the cookies it then shows the Tesla - SSO Login page again.

**Comment by paulobrien** [comment]
I have this too (also in Postman) and haven't sussed out why yet!

**Reply by infxmousrogue** [reply]
I figured it out. It is due to Postman automatically saving
your previous cookies. Delete all your cookies and try again from step 1. It should work.
